src/otx/algo/utils/xai_utils.py

```diff
@@ -37,13 +33,7 @@
     from otx.core.data.module import OTXDataModule
 
 ProcessedSaliencyMaps = list[dict[str, np.ndarray | torch.Tensor]]
-OTXBatchPredEntitiesSupportXAI = (
-    MulticlassClsBatchPredEntity
-    | MultilabelClsBatchPredEntity
-    | HlabelClsBatchPredEntity
-    | DetBatchPredEntity
-    | InstanceSegBatchPredEntity
-)
+OTXBatchPredEntitiesSupportXAI = TorchPredItem | DetBatchPredEntity | InstanceSegBatchPredEntity
```

Comment:
I guess we are going to handle this later, right? All of these names look verbose and intimidating to me

Reply:
Ideally we want to remove all the task specific entities